index.js

```diff
@@ -63,6 +64,21 @@ class ReadStream extends Readable {
       this.push(null)
       return cb(null)
     }
+    if (this.batch > 1) {
```

Comment:
this needs to check if feed.getBatch exists also (remotehypercore does not have this atm)

Reply:
It would be good to test the streams against remotehypercore as well?

Reply:
We already test it in hyperspace. This should only test against the "source of truth" which is hypercore :)

Reply:
How about a PR to remote-hypercore that adds getBatch?